roles/php/vars/Debian-bookworm.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+php_webserver: "apache2"
+
+php_default_packages:
+  - "libapache2-mod-php8.2"
+
+php_default_conf_scan_dir: "/etc/php/8.2/apache2/conf.d"
+
+# Only when php_alt_repo is set to true
+php_apt_repositories:
+  - "deb https://packages.sury.org/php/ bookworm main"
+
+php_apt_key: "https://packages.sury.org/php/apt.gpg"
+
+php_apt_packages:
+  - apt-transport-https
+  - ca-certificates
```

Comment:
This change should be in a separate PR which adds support for Debian bookworm to the php role

Reply:
ok